### DISCUSSION TITLE
Privacy issues with using this in org mode

### DISCUSSION THREAD

**Discussion by 1over137** [karthink/gptel#458] [General]
I'm new to emacs. If I use gptel-send in a narrowed buffer, does it send the whole file or just the visible part of the buffer?

**Comment by karthink** [comment]
> I'm new to emacs. If I use gptel-send in a narrowed buffer, does it send the whole file or just the visible part of the buffer?

Just the visible part, and just whatever is before the cursor.  If a region is selected, then it's just the region instead.
